content/blogposts/2023/cody-vscode-0-14-release.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "Cody for VS Code v0.12 release"
+authors:
+  - name: Kalan Chan
+    url: https://handbook.sourcegraph.com/team/#kalan-chan
+  - name: Tim Lucas
+    url: https://handbook.sourcegraph.com/team/#tim-lucas
+publishDate: 2023-10-04T10:00-07:00
+description: "Smart selection for `/doc` and `/test`, Inline Documentation with /doc, New custom command modes, New code block actions in chat"
```

Comment:
Just plain sentence case here (matching the previous posts) with a full stop. I've updated to match the suggested heading edits below.

```suggestion
description: "Smart selection for /doc and /test, improvements to /doc, a new mode configuration for custom commands, and updated code block action buttons in chat."
```